byteslice/byteslice.gobra

```diff
@@ -33,6 +33,15 @@ func Byte(s []byte, start int, end int, i int) byte {
 	return unfolding acc(Bytes(s, start, end), _) in s[i]
 }
 
+ghost
+requires acc(Bytes(ub, 0, len(ub)), _)
+ensures len(res) == len(ub)
+ensures forall i int :: { res[i] } 0 <= i && i < len(ub) ==>
+    res[i] == Byte(ub, 0, len(ub), i)
+decreases len(ub)
+opaque
+pure func View(ub []byte) (res seq[byte])
```

Comment:
To make sure that we are not missing anything, could we provide a body to this function here?